### PR TITLE
MS-157 Login form updates

### PR DESCRIPTION
**Pull request by luhmirin-s**
Main changes: 
* Removed unnecessary "hide password" toggle for user ID field
* Made project ID field read-only
* Made disabled input fields more obviously disabled
* Moved QR project ID comparison to the one provided by intent to happen right after the scan
* Added an option to manually change the backend URL for cases when QR code cannot be used for the login (extreme edge case, but still a valid one)

### CODE REVIEW

**Review by alexandr-simprints — APPROVED** [review]

File: infra/network/src/test/java/com/simprints/infra/network/url/BaseUrlProviderImplTest.kt [hunk 1]
```diff
@@ -14,7 +14,10 @@ import org.junit.Test
 class BaseUrlProviderImplTest {
 
     companion object {
+        private const val URL_SUFFIX = "/androidapi/v2/"
+
         private const val URL = "https://test"
+        private const val URL_WITH_PREFIX = "https://test$URL_SUFFIX"
```
Comment (alex-vt) [inline infra/network/src/test/java/com/simprints/infra/network/url/BaseUrlProviderImplTest.kt:20]: Was the intended name `URL_WITH_SUFFIX`?

**Review by alex-vt — APPROVED** [review]

File: infra/network/src/test/java/com/simprints/infra/network/url/BaseUrlProviderImplTest.kt (hunk 1 above)
Comment (luhmirin-s) [reply infra/network/src/test/java/com/simprints/infra/network/url/BaseUrlProviderImplTest.kt]: Yes, fixed.
